content/en/blog/_posts/2024-03-19-go-workspaces.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: blog
+title: 'Using Go workspaces in Kubernetes'
+date: 2024-03-19
```

Comment:
@thockin: When would you like this published? Is there a particular time during the day that is good for publication?

Reply:
No particular.